Sites/index.html

```diff
@@ -0,0 +1,138 @@
+                    title,
+                    "Cloud Gate" after the way it reflects the sky.</figcaption>
+
+
+                <h3> Hour 5 [ 2:02 ] </h3> <img src="img/Hour-5.jpg"
+                    alt="2 girls under a street sign titled 'Magnificent Mile'">
+                <figcaption> In the afternoon we walked the Magnificant Mile and did some window
+                    shopping. Because we couldn't bring any liquids on the plane, every morning we stopped by the
+                    Macy's perfume counter and just
+                    laid it on thick! The ladies working there started getting really annoyed by the end of the
+                    week.</figcaption>
+
+
+                <h3> Hour 6 [ 3:40 ] </h3> <img src="img/Hour-6.jpg"
+                    alt="two girls on bicycles with city skyline in the back">
+                <figcaption> Mid-day, we rented bicycles and rode by the Navy Pier which features a
+                    beautiful view of the skyline and Lake Michigan. I tried to take pictures while riding my
+                    bike and I literally fell off. There is also no railing between the sidewalk and Lake
+                    Michigan ! A
+                    very dangerous situation for uncoordinated people like myself.<div class="a"> The weather
+                        this day was surprisingly warm! It was the first day since Autumn that the lake had been
+                        defrosted. The water was so clear and bright which made it reflect the skyline
+                        beautifully.</div>
+                </figcaption>
+
+
+                <h3> Hour 7 [ 4:37 ] </h3> <img src="img/Hour-7.jpg"
+                    alt="selfie of three girls holding a basket of fries and a hotdog">
+                <figcaption> As a snack, we stopped by a foodtruck to get cheese fries and a
+                    Chicago-style hotdog ! T'was delicious.</figcaption>
+
+
+                <h3> Hour 8 [ 5:22 ] </h3> <img src="img/Hour-8.jpg"
+                    alt="picture of a window with words written in it's dust">
+                <figcaption> We took the pinkline on the above ground train to a neighboorhood called Pilson for some
+                    thrift and vintage shoppping. In this photo, someone had wrote "old werld" in the dust on a
+                    store window. I thought it was funny, thus, a picture was born.
+                </figcaption>
+
+
+                <h3> Hour 9 [ 6:41 ] </h3> <img src="img/Hour-9.jpg" alt="picture of the corner of a library bookshelf">
+                <figcaption> Nearby, we stopped into a bookstore. There were techinically no phones
+                    allowed for some reason but I was feling rebellious so this is a snapshot I took of the
+                    mystery
+                    section ! </figcaption>
+
+
+                <h3>Hour 10 [ 7: 10 ] </h3> <img src="img/Hour-10.jpg" alt="girl smiling outside of a bookshop window"
+                    .>
```

Comment:
This period is outside of the quotemarks and creates broken HTML:
```suggestion
                    >
```